docs/user/convert/function_usage.rst

```diff
@@ -300,6 +300,24 @@ Example: Parsing GeoJson geometry
         "geometry": {"type": "Point", "coordinates": [55, 56]}
     }
 
+reproject
+^^^^^^^^^
+
+Description: Project a geometry from its native CRS to EPSG:4326
+
+Usage: ``reproject($0,'EPSG:3857')``
+
+Example: Reprojecting a parsed point from EPSG:3857 to EPSG:4326:
```

Comment:
I think it'd be worth looking at the semantics around other reproject methods.  Since other systems don't awesome 4326, they might do things a little differently; I'm wondering if that difference will cause trouble/confusion.

Reply:
do you know some other projects I could look at? Main thing I've seen is GDAL, but that's command line so the semantics don't really translate well...

Reply:
From GeoTools there is this: http://docs.geotools.org/stable/userguide/library/referencing/crs.html.  

I suppose I'm most uncomfortable with the fact that for most reprojections, a user has to give a source and target CRS.  Since GeoMesa only does 4326, there is a clear target.  

I suppose I'm looking for a way to reduce a user's surprise at GeoMesa's insistence on 4326.:)

Reply:
maybe we should change the function name to something like `projectFrom` ? I didn't want to allow it to project from/to any CRS, since that would likely just cause more confusion... I can also add a note here that geomesa only supports 4326.